client/src/constants.js

```diff
@@ -1,3 +1,28 @@
 export const rolesAvailable = ['Admin', 'General'];
-
 export const apiUrl = process.env.NEXT_PUBLIC_API_URL;
+export const carouselContent = [
+  {
+    title: 'Monitor and Manage your Inventories',
+    imgName: 'inventory.jpg',
+  },
+  {
+    title: 'Track your Shipments',
+    imgName: 'transport.jpg',
+  },
+  {
+    title: 'View your Orders',
+    imgName: 'vendor.jpg',
+  },
+  {
+    title: "Analyze and View your Company's Financial",
```

Comment:
```suggestion
    title: "Analyze and View your Company's Finances",
```
What do you think of this?